dev/packages/beats/mysql-0.0.1/dataset/error/fields/fields.yml

```diff
@@ -1,17 +1,13 @@
 - name: mysql.error
   type: group
-  description: >
+  description: |
     Contains fields from the MySQL error logs.
   fields:
-    - name: thread_id
-      type: alias
-      path: mysql.thread_id
-      migration: true
-    - name: level
-      type: alias
-      path: log.level
-      migration: true
-    - name: message
-      type: alias
-      path: message
-      migration: true
+  - name: level
```

Comment:
@mtojek I think I don't fully understand the change. Why is thread_id removed but not level here?

Reply:
As the PR title suggests I removed all fields marked with "migration" which are not ECS fields. In the other PR, you asked for removing it: https://github.com/elastic/package-registry/pull/323#discussion_r402241871

If I should remove all fields with `migration: true`, don't we need ecs.yml in every fields folder? As we don't have it there, I left ECS fileds in place.

Reply:
We need to remove all migration fields, also the ones that point to ECS fields.

We will need a subset of the ecs.yml field in most datasets, but not all of ecs fields. We only need the fields from ECS that are actually used by the dataset. For example if `log.level` is used, it needs to be in, but if we don't use source.ip, it shouldn't be.

Reply:
I've updated the PR.

Used ECS fields are stored in dedicated `ecs.yml`. Also, all used fields are in corresponding README files.